codec/decoder/plus/src/welsDecoderExt.cpp

```diff
@@ -744,7 +744,7 @@ void CWelsDecoder::OutputStatisticsLog (SDecoderStatistics& sDecoderStatistics)
 
 DECODING_STATE CWelsDecoder::ReorderPicturesInDisplay (unsigned char** ppDst, SBufferInfo* pDstInfo) {
   if (pDstInfo->iBufferStatus == 1 && m_pDecContext->pSps->uiProfileIdc != 66
-      && m_pDecContext->pPps->bEntropyCodingModeFlag) {
+      /*&& m_pDecContext->pPps->bEntropyCodingModeFlag*/) {
```

Comment:
I think it would be better to delete the bEntropyCodingModeFlag check entirely. It's just clutter now.